app/javascript/components/schedule-form/schedule-form-fields.js

```diff
@@ -0,0 +1,276 @@
+import { componentTypes, validatorTypes } from '@@ddf';
+import {
+  actionChange, runOptionChange, runOptions, subActionChange, objectTypeChange, restructureOptions, runAtChange,
+} from './helper';
+
+export const attributeValueLimit = 5;
+
+export const nameField = () => ({
+  component: componentTypes.TEXT_FIELD,
+  name: 'name',
+  label: __('Name'),
+  maxLength: 50,
+  isRequired: true,
+  validate: [{ type: validatorTypes.REQUIRED }],
+  autoFocus: true,
+});
+
+export const descriptionField = () => ({
+  component: componentTypes.TEXTAREA,
```

Comment:
This is more a question but is there a reason this was changed to a text area? Previously it was just a text-box.
![image](https://user-images.githubusercontent.com/64800041/217947809-14e5d788-e166-4973-a053-10539a501290.png)

Reply:
Reverted to text field like the previous form.